plugins/github-deployments/src/api/index.ts

```diff
@@ -28,21 +28,22 @@ const getBaseUrl = (
   }
 
   const location = parseLocationReference(host);
-  if (location.type !== 'github') {
+  if (location.type !== 'github' && location.type !== 'url') {
     return 'https://api.github.com';
   }
 
-  const config = scmIntegrationsApi.github.byHost(location.target);
+  const hostname = new URL(location.target).hostname;
+  const config = scmIntegrationsApi.github.byHost(hostname);
```

Comment:
Oh but there is a byUrl method, no need for this manual parsing!

Reply:
thanks for the tip!